core/db/entities.py

```diff
@@ -572,6 +572,8 @@ class Submission(BaseModel):
     reporting_round = sqla.Column(sqla.Integer(), nullable=False)
     submission_filename = sqla.Column(sqla.String(), nullable=True)
     data_blob = sqla.Column(JSONB, nullable=True)
+    submitting_account_id = sqla.Column(sqla.String())
+    submitting_user_email = sqla.Column(sqla.String())
```

Comment:
We may want to think about whether we can backfill these columns and make them non-nullable in the future. Could you make a ticket for this and put it in the refinement bucket of one of the boards please (probably submit's board)? :)